.travis.yml

```diff
@@ -12,7 +17,7 @@ before_script:
 script:
 - |
   # get ourselves a MIR-ful libstd
-  xargo/build.sh
+  cd xargo && RUSTFLAGS='-Zalways-encode-mir' xargo build && cd ..
```

Comment:
Can you keep `xargo/build.sh` here? My point was that *if* the script can be made more portable, it could be used on macOS too.